index.html

```diff
@@ -27,6 +27,157 @@ <h2>I’m a software developer</h2>
+                </ul>
+                <button class="projects" type="button" onclick="projects">See projects</button>
+            </div>
+            <div>
+                <img class="img" src="plan.png" alt="planholder">
+                <h2>Multi-Post Stories Gain+Glory</h2>
+                <ul class="language">
+                    <li><button type="button" onclick="Ruby">Ruby on rails</button></li>
+                    <li><button type="button" onclick="css">css</button></li>
+                    <li><button type="button" onclick="Java">Java Script</button></li>
+                    <li><button type="button" onclick="html">HTML</button></li>
+                </ul>
+                <button class="projects" type="button" onclick="projects">See projects</button>
+            </div>
+            <div>
+                <img class="img" src="plan.png" alt="planholder">
+                <h2>Multi-Post Stories Gain+Glory</h2>
+                <ul class="language">
+                    <li><button type="button" onclick="Ruby">Ruby on rails</button></li>
+                    <li><button type="button" onclick="css">css</button></li>
+                    <li><button type="button" onclick="Java">Java Script</button></li>
+                    <li><button type="button" onclick="html">HTML</button></li>
+                </ul>
+                <button class="projects" type="button" onclick="projects">See projects</button>
+            </div>
+            <div>
+                <img class="img" src="plan.png" alt="planholder">
+                <h2>Multi-Post Stories Gain+Glory</h2>
+                <ul class="language">
+                    <li><button type="button" onclick="Ruby">Ruby on rails</button></li>
+                    <li><button type="button" onclick="css">css</button></li>
+                    <li><button type="button" onclick="Java">Java Script</button></li>
+                    <li><button type="button" onclick="html">HTML</button></li>
+                </ul>
+                <button class="projects" type="button" onclick="projects">See projects</button>
+            </div>
+            <div>
+                <img class="img" src="plan.png" alt="planholder">
+                <h2>Multi-Post Stories Gain+Glory</h2>
+                <ul class="language">
+                    <li><button type="button" onclick="Ruby">Ruby on rails</button></li>
+                    <li><button type="button" onclick="css">css</button></li>
+                    <li><button type="button" onclick="Java">Java Script</button></li>
+                    <li><button type="button" onclick="html">HTML</button></li>
+                </ul>
+                <button class="projects" type="button" onclick="projects">See projects</button>
+            </div>
+        </div>
+            </section>
+            <section class="AboutMeHeader">
+                <div class="resume">
+                <h1 class="me">About me</h1>
```

Comment:
- [ ] [Optional] You should use only one ```h1``` tag in your HTML file, please consider using another ```hx``` for this section. this is good for the better practices and SEO